open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerConfigOpenLineage.java

```diff
@@ -19,93 +20,103 @@
 public class OMAGServerConfigOpenLineage {
     private final OMAGServerAdminStoreServices configStore = new OMAGServerAdminStoreServices();
 
-    private static final String serviceName    = GovernanceServicesDescription.OPEN_LINEAGE_SERVICES.getServiceName();
+    private static final String serviceName = GovernanceServicesDescription.OPEN_LINEAGE_SERVICES.getServiceName();
     private static final String defaultALOutTopicName = "omas.omas.assetlineage.outTopic";
 
+
     private OMAGServerErrorHandler errorHandler = new OMAGServerErrorHandler();
     private OMAGServerExceptionHandler exceptionHandler = new OMAGServerExceptionHandler();
 
     /**
      * Set the Open Lineage Config
      *
-     * @param userId  user that is issuing the request.
-     * @param serverName  local server name.
-     * @param openLineageServerConfig  Config for the Open Lineage Services
+     * @param userId                  user that is issuing the request.
+     * @param serverName              local server name.
+     * @param openLineageServerConfig Config for the Open Lineage Services
      * @return void response
      **/
     public VoidResponse setOpenLineageConfig(String userId, String serverName, OpenLineageServerConfig openLineageServerConfig) {
+
         String methodName = "setOpenLineageConfig";
         VoidResponse response = new VoidResponse();
 
         try {
             errorHandler.validateServerName(serverName, methodName);
             errorHandler.validateUserId(userId, serverName, methodName);
 
+            if (openLineageServerConfig.getInTopicName() == null)
+                throwMissingPropertyException(serverName, "inTopicName");
```

Comment:
Can method _throwMissingPropertyException_ be written in the errorHandler, similar to validate* methods ? Ex: validateInTopicName() etc.
I am quite certain it can be done. Explore this possibility, see how the code changes

Reply:
Done, good suggestion